src/app/views/route-search-results/route-search-results.component.html

```diff
@@ -0,0 +1,45 @@
+<app-nav-bar title="Results"></app-nav-bar>
+
+<div class="big-navigation-gap">
+  <div class="route-search-results__route-information">
+    <div>
+      <div>A</div>
+      <div>B</div>
```

Comment:
Maybe from / to ?

Reply:
Or like you did in the previous-routes-component with the two circles and the connecting line?

Reply:
Yeah was thinking about - but I'm not sure about the size, but probably best idea as both route dialogues may look alike :)